src/DTO/Bandit/BanditVariation.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Eppo\DTO\Bandit;
+
+class BanditVariation
+{
+    public function __construct(
+        public readonly string $key,
```

Comment:
Since we are not relying on JSON automatic, I wonder if it would be better to explicitly call this `$banditKey`.

I know in other SDKs, we just call this `key,` so I'm fine either way, but I have a slight preference for `$banditKey` given the other keys also in this class.

Reply:
It's nice to have your keys labeled when you have a whole pile.
The reflection/parsing libraries used in the other SDKs usually have a means of specifying custom attribute names (python and .net do for sure). Something we can consider next time we're making changes there.